main_window/main_app.py

```diff
@@ -19,7 +18,7 @@
 class MainApp(QtWidgets.QMainWindow):
     sig_send = pyqtSignal(str)
 
-    def __init__(self, connections: Dict[str, Connection], rocket_profile: RocketProfile) -> None:
+    def __init__(self, connections: Dict[str, Connection], rocket_profile) -> None:
```

Comment:
Why was this change made?

Reply:
Import loop. MainApp tries to import RocketProfile but RocketProfile imports MainApp.